api/src/main/java/com/okta/sdk/resource/user/UserBuilder.java

```diff
@@ -18,7 +18,9 @@
 import com.okta.sdk.client.Client;
 import com.okta.sdk.lang.Classes;
 
+import java.util.List;
```

Comment:
Is `java.util.List` used anywhere in the `UserBuilder`?

Reply:
Nope, that should have failed CI... I fix that and figure out why that didn't get picked up by the static analysis